_posts/2019-03-06-release-1.8.0.md

```diff
@@ -0,0 +1,187 @@
+---
+layout: post
+title:  "Apache Flink 1.8.0 Release Announcement"
+date:   2019-2-26 12:00:00
+categories: news
+authors:
+- till:
+  name: "Aljoscha Krettek"
+  twitter: "aljoscha"
+---
+
+The Apache Flink community is pleased to announce Apache Flink 1.8.0.  The
+latest release includes more than 370 resolved issues and some exciting
+additions to Flink that we describe in the following sections of this post.
+Please check the [complete changelog](https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12315522&version=12344274)
+for more details.
+
+Flink 1.8.0 is API-compatible with previous 1.x.y releases for APIs annotated
+with the `@Public` annotation.  The release is available now and we encourage
+everyone to [download the release](http://flink.apache.org/downloads.html) and
+check out the updated
+[documentation](https://ci.apache.org/projects/flink/flink-docs-release-1.8/).
+Feedback through the Flink [mailing
+lists](http://flink.apache.org/community.html#mailing-lists) or
+[JIRA](https://issues.apache.org/jira/projects/FLINK/summary) is, as always,
+very much appreciated!
+
+You can find the binaries on the updated [Downloads page](http://flink.apache.org/downloads.html) on the Flink project site.
+
+{% toc %}
+
+With Flink 1.8.0 we come closer to our goals of enabling fast data processing
```

Comment:
"to our goals of enabling fast data processing and building data-intensive applications for the" -> 
"to our goal of providing a ready-to-use solution for fast data processing that enables the creation of data-intensive applications..."